cmd/root.go

```diff
@@ -92,6 +92,7 @@ func init() {
 	rootCmd.Flags().String("github-host", "", "Optional set host for GitHub Enterprise")
 	rootCmd.Flags().Bool("no-post-mode", false, "Optional do not post comment to VCS, instead write additional file and print diff to stdout")
 	rootCmd.Flags().Bool("disable-collapse", false, "Collapsible comments are enabled by default for GitHub and GitLab. When set to true it will not use collapsed sections.")
+	rootCmd.Flags().Bool("show-overview", false, "Show Overview are disabled by default. When set to true it will show an number of diffs and number of replaces in the overvoew section.")
```

Comment:
typo

Reply:
```suggestion
	rootCmd.Flags().Bool("show-overview", false, "Show Overview are disabled by default. When set to true it will show the number of cdk stacks with diff and  the number of replaced resources in the overview section.")
```

Reply:
updated